src/styles/admin-core/content-blocks/_cards-without-description.scss

```diff
@@ -0,0 +1,50 @@
+@use "sass:math";
+
+@use "src/styles/abstracts" as *;
+
+$component: "c-block-cards-without-description";
+$img-size: 15.2rem;
+
+.#{$component} {
+	width: 100%;
+	display: flex;
+	flex-flow: row wrap;
+	grid-gap: $spacer-lg;
+	justify-content: space-around;
+
+	&__card {
+		height: $img-size;
+		width: 38.4rem;
+		color: $white;
+		display: grid;
+		grid-template-columns: repeat(2, 7.6rem) 23.2rem;
+	}
+
+	&__title {
+		grid-column: 2/4;
+		grid-row: 1;
+		display: flex;
+		align-items: center;
+		padding: 0 $spacer-md 0 calc(math.div($img-size, 2) + $spacer-md);
+		font-weight: $font-weight-bold;
+		font-size: 2rem;
+		background-color: $black;
+	}
+
+	&__image {
+		width: $img-size;
+		height: $img-size;
+		overflow: hidden;
+		grid-column: 1/2;
+		grid-row: 1;
+
+		img {
+			width: 100%;
+			height: 100%;
+		}
```

Comment:
Non square images are currently being stretched, we can avoid this with the object-fit property: https://www.w3schools.com/css/css3_object-fit.asp

![image](https://user-images.githubusercontent.com/1710840/232739776-09812924-19eb-4417-b7ed-200edd71357f.png)

```suggestion
		img {
			width: 100%;
			height: 100%;
			object-fit: cover;
		}
```